src/main.rs

```diff
@@ -95,7 +96,9 @@ fn run() -> i32 {
     };
 
     let window_rect = display.get_window_rect(window);
-    let sel = match matches.opt_str("g") {
+
+    let mut sel: util::Rect;
+    sel = match matches.opt_str("g") {
```

Comment:
I don't think splitting this line in two or even specifying the type is strictly necessary here. Is it?